README.md

```diff
@@ -1055,15 +1055,17 @@ yargs have been validated.
 
 Method to execute when a failure occurs, rather than printing the failure message.
 
-`fn` is called with the failure message that would have been printed and the
-`Error` instance originally thrown, if any.
+`fn` is called with the failure message that would have been printed, the
+`Error` instance originally thrown and yargs state when the failure
```

Comment:
slight editing nit, let's change this to:

`fn` is called with the failure message that would have been printed, the
`Error` instance originally thrown, and the yargs instance when the failure occurred.